configs/accengage.json

```diff
@@ -1,30 +1,41 @@
 {
   "index_name": "accengage",
   "start_urls": [
-    "https://docs-alpha.accengage.com"
+    "https://documentation.accengage.com",
+    "https://documentation.accengage.com/sdk",
+    "https://documentation.accengage.com/api",
+    "https://documentation.accengage.com/dataflow/introduction",
+    "https://documentation.accengage.com/user-guide/en",
+    "https://documentation.accengage.com/user-guide/fr",
+    {
+      "url": "https://documentation.accengage.com/faq",
+      "selectors_key": "faq"
+    }
   ],
-  "stop_urls": [],
+  "stop_urls": ["https://documentation.accengage.com/admin"],
   "selectors": {
-    "lvl0": {
-      "selector": "(//nav//a[contains(@class, 'active')])[1]/../../preceding::a[1]",
-      "type": "xpath",
-      "global": true,
-      "default_value": "Documentation"
+    "default": {
+      "lvl0": "h1, #title-text",
+      "lvl1": "h2",
+      "lvl2": "h3",
+      "lvl3": "h4",
+      "lvl4": "h5",
+      "text": "p, li, ul, ol"
     },
-    "lvl1": {
-      "selector": "(//nav//a[contains(@class, 'active')])[1]",
-      "type": "xpath",
-      "global": true,
-      "default_value": "Chapter"
-    },
-    "lvl2": "main h2",
-    "lvl3": "main h3",
-    "lvl4": "main h4",
-    "text": "main p, main li"
+    "faq": {
+      "lvl0": "p, li, h1, #title-text",
+      "lvl1": "h2",
+      "lvl2": "h3"
```

Comment:
You still need to define the common selectors from the `default` object that you want to be applied.

Reply:
To define a common selector i must apply the global key to true, or there's another way?

Reply:
No, defining a global selector will impact the way we process the elements matching global selectors. It is not related to such use case. [See here](https://community.algolia.com/docsearch/recommended-configuration.html#recommended-selectors) for more details.

You will need to duplicate the common selectors from one object to the other.